querydsl-apt/src/main/java/com/querydsl/apt/DefaultConfiguration.java

```diff
@@ -177,6 +181,19 @@ public DefaultConfiguration(
             }
         }
 
+        if (options.containsKey(QUERYDSL_VARIABLE_NAME_FUNCTION_CLASS)) {
+            try {
+                variableNameFunction = (Function<EntityType, String>) Class.forName(options.get(QUERYDSL_VARIABLE_NAME_FUNCTION_CLASS)).newInstance();
```

Comment:
If you split this line up to something like this:
```java
@SuppressWarnings("unchecked")
Class<Function<EntityType, String>> variableNameFunctionClass
        = (Class<Function<EntityType, String>>) Class.forName(options.get(QUERYDSL_VARIABLE_NAME_FUNCTION_CLASS));
variableNameFunction = variableNameFunctionClass.newInstance();
```

Then that means not the entire constructor carries the `@SuppressWarnings` anymore.